configure.ac

```diff
@@ -548,12 +546,41 @@ if test -z "$ENABLE_SOS_TRUE"; then
 	fi
 fi
 
-dnl ldms-python API's
+dnl ldms-python API's and programs
+AX_WITH_PROG([CYTHON], [cython], [no])
+CYTHON_REQ_VER="0.28.5"
+CYTHON_MSG="
+
+    Please install Cython ver >= ${CYTHON_REQ_VER}. It is recommended to install
+    Cython via pip3 (\`pip3 install Cython\`) to get the latest version of
+    Cython.  Some installation package (e.g. python36-Cython from epel) may
+    install cython program with cython<VER> form (e.g. /bin/cython3.6). In such
+    case, --with-cython=FULL_PATH_TO_CYTHON_PROGRAM option is required (e.g.
+    --with-cython=/bin/cython3.6).
+
```

Comment:
This message isn't accurate. The code below is checking for cython3.6. I'm not sure this comment is needed either. "--with" is pretty standard for specifying a custom path to something, and pip3 is not at all a universally recommended way to handle the installation. It probably isn't the way many HPC installations would do it.

Reply:
The code below additionally looks for cython3 or cython3.6 so that on Unbuntu 18.04 (the one use for CI/CD script) and on RHEL7 we can `./configure` without having to specify `--with-cython=/usr/bin/cython3` or `--with-cython=/bin/cython3.6` respectively.